config/policies.txt

```diff
@@ -1,5 +0,0 @@
-## SERVER POLICIES ##
-# Each line is pure html that gets sent to the user under certain conditions
-
-# When a mob is polymorphed
-POLYMORPH <span class='danger'>Note that you are allowed to act as an antagonist while transformed into a hostile mob, unless you volunteered for or sought out transformation.</span>
```

Comment:
I don't see this anywhere am I missing something?

Reply:
No, i just didn't add it to default file since it was broken before and no one complained anyway. Headmins can add it back if they want to.

Reply:
👍